src/ux/confirmInformation/ConfirmInformation.tsx

```diff
@@ -58,23 +63,36 @@ export function ConfirmInformation ({ displayMode, chainId, address, providerUse
 
       <List>
         <Column>
-          <Title><Trans>Wallet address</Trans>:</Title>
+          <Description>{shortAddress(address)}</Description>
+          <Title><Trans>Wallet</Trans>:</Title>
           {peerWallet && <Title><Trans>Connected wallet</Trans>:</Title>}
           <Title><Trans>Network</Trans>:</Title>
           {isHardwareWallet && <Title><Trans>Derivation path</Trans>:</Title>}
           {sd && Object.keys(sd.claims).map(key => <Title key={`claim-key-${key}`}>{key}:</Title>)}
           {sd && Object.keys(sd.credentials).map(key => <Title key={`credential-key-${key}`}>{key}:</Title>)}
         </Column>
         <Column>
-          <Description>{shortAddress(address)}</Description>
+          <Title>
+            <Clickable onClick={() => navigator.clipboard.writeText(address!)}><Trans>copy</Trans> <img src={CopyIcon} alt="copy" /></Clickable>
+            {infoOptions[chainId!] && <>
+              &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+              <Clickable onClick={() => window.open(`${infoOptions[chainId!].addressBaseURL}${address}`, '_blank')!.focus()}><Trans>explorer</Trans> <img src={LinkIcon} alt="explorer" /></Clickable>
+            </>}
```

Comment:
`&nbsp;` should not be used for placement. If you need to create space use padding on a `span` tag

Reply:
There is another like that in 
```tsx
{!displayMode && <Header2><img src={SuccessIcon} alt="success" /> <Trans>Successfully connected</Trans></Header2>}
```

will fix that too